qcelemental/models/molecule.py

```diff
@@ -67,21 +69,17 @@ class Identifiers(BaseModel):
     canonical_isomeric_smiles: Optional[str] = None
     canonical_smiles: Optional[str] = None
 
-    class Config:
-        allow_mutation = False
-        extra = "forbid"
-
-    def dict(self, *args, **kwargs):
-        return super().dict(*args, **{**kwargs, **{"skip_defaults": True}})
+    class Config(ProtoModel.Config):
+        serialize_skip_defaults = True
 
 
-class Molecule(BaseModel):
+class Molecule(ProtoModel):
```

Comment:
So this change will make previous versions of Molecule incompatible due to the changes to List->Array (I think). Just a note for the change log more than anything.

Reply:
Previous versions will upcast seamlessly because of the `np.asarray`.